### PR TITLE
Implements issue 51

### PR DESCRIPTION
**Pull request by ChenJesse**
If the user has no posts, only shows the current month. Else, the calendar only goes back as far as his/her earliest post

### CODE REVIEW

**Comment by ChenJesse** [comment]
You're right, I think I fixed it...also I think the "loading" that we thought yesterday was coming from the ProfileViewController seems to be more of a hit detection/clicking problem

File: IceFishing/Profile/ProfileViewController.swift [hunk 1]
```diff
@@ -207,18 +213,23 @@ class ProfileViewController: UIViewController, UICollectionViewDelegate, UIColle
...
-            numberOfDays = startDate.numDaysUntilEndDate(firstDayOfMonth.lastDayOfMonth())
-        }
-        
-        return numberOfDays
+		var firstDayOfMonth: NSDate
+		if let date = earliestPostDate {
+			firstDayOfMonth = date.firstDayOfMonth().dateByAddingMonths(numOfSections! - section - 1)
+		}
+		else {
+			firstDayOfMonth = NSDate().firstDayOfMonth()
```
Comment (lucasderraugh) [inline IceFishing/Profile/ProfileViewController.swift]: Is there a reason to get the `firstDayOfMonth()` and then get the `numDaysInMonth()`?

**Comment by lucasderraugh** [comment]
@ChenJesse If you could check these changes and see if you agree with them, that would be swell :+1:. Then you're good to :ship: it.

**Comment by lucasderraugh** [comment]
You're right. I was thinking for some reason earliestPostDate was the current date. Couldn't we simplify this then to just use the current date instead? We can resume this conversation on Slack though.

**Pull request closed without merging**